src/lib.rs

```diff
@@ -40,7 +40,7 @@
     rustdoc::broken_intra_doc_links
 )]
 #![forbid(unsafe_code)]
-#![doc(html_root_url = "https://docs.rs/rusty-s3/0.1.2")]
+#![doc(html_root_url = "https://docs.rs/rusty-s3/0.2.0")]
```

Comment:
FYI, the rust API guidelines no longer recommend setting `html_root_url`. See https://github.com/rust-lang/api-guidelines/pull/230.